.github/workflows/continuous-integration.yml

```diff
@@ -125,6 +125,9 @@ jobs:
       - run: npm ci
         working-directory: frontend
 
+      - run: npm run build
```

Comment:
Ich würde das schlussendlich eher in einem eigenen job machen, sodass die unit tests weiterhin schnell failen. Aber im Moment lieber wieder den Build zum laufen bringen :+1: